packages/psss-server/src/routes/reports/SubmitConfirmedCountryWeeklyReportRoute.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { RespondingError } from '@tupaia/utils';
+import { Route } from '../Route';
+import { PSSS_WEEKLY_REPORT_CODE } from '../../constants';
+
+const SURVEY_CODE = 'PSSS_Confirmed_WNR';
+
+type ConfirmedWeeklyReportAnswers = {
+  PSSS_Confirmed_Sites: number;
+  PSSS_Confirmed_Sites_Reported: number;
+  PSSS_Confirmed_AFR_Cases: number;
+  PSSS_Confirmed_DIA_Cases: number;
+  PSSS_Confirmed_ILI_Cases: number;
+  PSSS_Confirmed_PF_Cases: number;
+  PSSS_Confirmed_DLI_Cases: number;
+};
+
+export class SubmitConfirmedCountryWeeklyReportRoute extends Route {
+  async buildResponse() {
+    const { week } = this.req.query;
+    const { organisationUnitCode } = this.req.params;
+
+    const report = await this.reportConnection?.fetchReport(
+      PSSS_WEEKLY_REPORT_CODE,
+      [organisationUnitCode],
+      [week],
+    );
```

Comment:
We fetch the existing unconfirmed data from the report-server here. We could actually construct a separate report for this that only returns the required data? It should have improved performance that way...

Reply:
Oh or the front end could just send through the data in the request body?

Reply:
We talked about this and confirmed that requesting the data here again is safest, though might be made faster by the addition of another report. Happy to resolve this comment